src/test/java/seedu/address/commons/util/AppUtilTest.java

```diff
@@ -1,17 +1,11 @@
 package seedu.address.commons.util;
 
-import static org.junit.jupiter.api.Assertions.assertNotNull;
 import static seedu.address.testutil.Assert.assertThrows;
 
 import org.junit.jupiter.api.Test;
 
 public class AppUtilTest {
 
-    @Test
```

Comment:
Can I check what is the reason for deleting this block of code?

Reply:
There was a specific error that only my computer had, so i removed it to be able to commit my branch!